### PR TITLE
docs: add backend configuration to metasrv doc

### PR DESCRIPTION
**Pull request by lyang24**
## What's Changed in this PR
similar to https://github.com/GreptimeTeam/greptimedb/pull/4573 This pr adds examples to the backend field in metasrv config in the website 

*Describe the change in this PR*

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

**Review by killme2008 — APPROVED** [review]

**Review by nicecui — COMMENTED** [review]
please also update the content at `docs/i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/configuration.md`

**Comment by lyang24** [comment]
> please also update the content at `docs/i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/configuration.md`

Done thanks.

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/configuration.md [hunk 1]
```diff
@@ -499,7 +499,7 @@ data_home = "/tmp/metasrv/"
 bind_addr = "127.0.0.1:3002"
 # frontend 和 datanode 连接到 metasrv 的通信服务器地址，本地默认为 "127.0.0.1:3002"。
 server_addr = "127.0.0.1:3002"
-# Etcd 服务器地址，默认为 "127.0.0.1:2379"。
+# metasrv存储端服务器地址，默认为 "127.0.0.1:2379"。
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/configuration.md:502]:
```suggestion
# metasrv 存储端服务器地址，默认为 "127.0.0.1:2379"。
```

**Review by nicecui — APPROVED** [review]

**Comment by nicecui** [comment]
waiting for the merge of https://github.com/GreptimeTeam/greptimedb/pull/4573

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/configuration.md [hunk 2]
```diff
@@ -513,6 +513,8 @@ use_memory_store = false
 ## - 使用 Remote WAL
 ## - 使用共享存储（例如 s3）。
 enable_region_failover = false
+# metasrv的数据库类型.
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/configuration.md:516]:
```suggestion
# metasrv 的数据库类型.
```

**Pull request merged**